lib/livebook/file_system/local.ex

```diff
@@ -302,4 +302,6 @@ defimpl Livebook.FileSystem, for: Livebook.FileSystem.Local do
   def load(_file_system, _fields), do: raise("not implemented")
 
   def dump(_file_system), do: raise("not implemented")
+
+  def credentials(_file_system), do: raise("not implemented")
```

Comment:
Maybe we should use `dump` instead of adding a new callback? Thoughts?

Reply:
Wouldn't we add duplicated fields in this case?

Reply:
Which duplicate fields?

Reply:
It is fine to duplicate because we should not couple them. :)